src/lib/TwaGenerator.js

```diff
@@ -158,15 +158,16 @@ class TwaGenerator {
     };
 
     console.log('Generating Android Project files:');
-    const templateDirectory = path.join(__dirname, '../template_project');
+    console.log(__dirname);
+    const templateDirectory = path.join(__dirname, '../../template_project');
 
     const copyFileList = new Set(COPY_FILE_LIST);
     if (!args.maskableIconUrl) {
       DELETE_FILE_LIST.forEach((file) => copyFileList.delete(file));
     }
 
     // Copy Project Files
-    await this._copyStaticFiles(templateDirectory, targetDirectory, [...copyFileList]);
+    await this._copyStaticFiles(templateDirectory, targetDirectory, Array.from(copyFileList));
```

Comment:
Glad you fixed this, too. After getting around some of the initial hurdles, I ran into this one, where TypeScript's __spreadArrays helper doesn't work on a Set. Thanks for fixing this.